js/components/controlBar.js

```diff
@@ -129,10 +131,13 @@ var ControlBar = React.createClass({
           this.props.controller.toggleFullscreen();
         }
 
-        if (this.vr.stereo) {
+        if (this.props.controller.isVrStereo) {
+          this.props.controller.checkDeviceOrientation = true;
           this.setLandscapeScreenOrientation();
+          window.addEventListener("orientationchange", this.setLandscapeScreenOrientation, false);
```

Comment:
This might not be a good place to add a listener.  It looks like the listener will never get removed if the button isn't clicked again. I think it would be better to set the listener (for vr only) on mount (and remove on unmount) and then determine whether to call `this.setLandscapeScreenOrientation` or not depending on the condition

Reply:
\+